vulnerability-classes/injection/Detecting hidden input fields for XSS.bcheck

```diff
@@ -1,17 +1,53 @@
 metadata:
-    language: v1-beta
+    language: v2-beta
     name: "Hidden Input Field Detection at Response Level (Passive)"
-    author: "mrrootsec"
-    description: "This Bcheck identifies hidden input fields in the response for further examination."
-    tags: "passive" , "xss"
+    author: "mrrootsec, Kyle Gilligan"
+    description: "This Bcheck identifies hidden input fields in the HTTP response for further examination."
+    tags: "passive", "xss", "html", "hidden"
+
+define:
+
+    # Issue Details:
```

Comment:
Splitting the issue details and remediation advice into small chunks makes this code difficult to read. Can you please inline these into the `report issue:` block.